packages/expo-updates/ios/EXUpdates/EXUpdatesModule.m

```diff
@@ -48,7 +48,9 @@ - (NSDictionary *)constantsToExport
       @"releaseChannel": _updatesService.config.releaseChannel,
       @"localAssets": _updatesService.assetFilesMap ?: @{},
       @"isEmergencyLaunch": @(_updatesService.isEmergencyLaunch),
-      @"isMissingRuntimeVersion": @(_updatesService.config.isMissingRuntimeVersion)
+      @"isMissingRuntimeVersion": @(_updatesService.config.isMissingRuntimeVersion),
+      @"runtimeVersion": _updatesService.config.runtimeVersion ?: @"",
+      @"channel": _updatesService.config.requestHeaders[@"expo-channel-name"] ?: @""
```

Comment:
we should probably add these to the objects on lines 29/37 as well, in order avoid future issues like https://github.com/expo/expo/pull/15451 (or document that these properties are only available in release builds).